superset/viz.py

```diff
@@ -871,15 +895,10 @@ def get_data(self, df: pd.DataFrame) -> VizData:
         groupby = self.form_data.get("groupby") or []
         columns = self.form_data.get("columns") or []
 
-        def _format_datetime(value: Any) -> Optional[str]:
-            if isinstance(value, str):
-                return f"__timestamp:{datetime_to_epoch(pd.Timestamp(value))}"
-            return None
-
         for column_name in groupby + columns:
             column = self.datasource.get_column(column_name)
-            if column and column.type in ("DATE", "DATETIME", "TIMESTAMP"):
-                ts = df[column_name].apply(_format_datetime)
+            if column and column.is_temporal:
```

Comment:
Without calling `column.is_temporal`, Postgres datasources were not able to identify temporal columns e.g. `TIMESTAMP WITHOUT TIMEZONE`.